pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -190,6 +190,21 @@ func toKubeRuntimeStatus(status *runtimeapi.RuntimeStatus) *kubecontainer.Runtim
 	return &kubecontainer.RuntimeStatus{Conditions: conditions}
 }
 
+// toKubeRuntimeConfig converts the runtimeapi.ActiveRuntimeConfig to kubecontainer.RuntimeConfigInfo
+func toKubeRuntimeConfig(config *runtimeapi.ActiveRuntimeConfig) *kubecontainer.RuntimeConfigInfo {
+	userNSConfig := kubecontainer.UserNamespaceConfigInfo{
+		UidMapping: kubecontainer.UserNSMapping{
+			ContainerID: config.UserNamespaceConfig.UidMappings[0].ContainerId,
```

Comment:
can UidMappings contain 0 items? can it contain more than 1 item? This seems like it is losing info

Reply:
contain 0 items --- yes, should have retrieved like config.GetUserNamespaceConfig().GetUidMappings(). will fix. Thanks!

contain more than 1 item ---- we are interested in only ID on host for ID 0 in container, therefore we are taking first ID of first mapping.